### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by yichen-fjy** [community/community#46231] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, my github account @zonemeen was suspended few days ago without any warning or notification mail. Can I get a proper reason behind it? Because I lost all my codes and repositories. I have sent four e-mails to the support team, but I haven't received a response yet, what should I do if anyone has encountered this situation, thanks

**Comment by jge162** [comment]
You can discuss the issue but we are unable to resolve for you here sorry. U must address with GitHub support.

**Reply by yichen-fjy** [reply]
> You can discuss the issue but we are unable to resolve for you here sorry. U must address with GitHub support.

Thanks for answering. I already sent them tickets four days ago. However, I received no response from them. To be clear, this is my friend's github account.

**Reply by jge162** [reply]
Yeah he has to wait for a response from support. we are only an avenue for discussion here. We cannot intact changes to accounts or reenable them sorry.  Support is your best help.

**Comment by yichen-fjy** [comment]
Hi @queenofcorgis, My friend @zonemeen already sent them tickets a week ago to apply for account recovery. However, he still haven't received a response from them. Do you know what's going on? Thanks

**Reply by queenofcorgis** [reply]
Hi there! Could you give me the ticket number please? I'm not able to escalate or interfere, but I can confirm that it is in the right place.

**Reply by yichen-fjy** [reply]
> Hi there! Could you give me the ticket number please? I'm not able to escalate or interfere, but I can confirm that it is in the right place.

Thanks for answering. The ticket number: 1992803

**Reply by queenofcorgis** [reply]
Can confirm it is in the right place, our tickets are answered in the order they were received so I appreciate all the patience.

**Reply by yichen-fjy** [reply]
> Can confirm it is in the right place, our tickets are answered in the order they were received so I appreciate all the patience.

According to previous cases, how long does my friend usually have to wait? Two weeks, or will it take longer? Because this account is very, very important for my friend. Thank you very much for your patient reply!

**Reply by ahmedivy** [reply]
Bro same happened to me like 3 months ago. They suspended my account without any warning or mail. I contacted them on support, and they reply after almost 40 days and says that your name matches some name on US Sanctions List (I was WT* 😂) and then they told me to provide picture of national identity card etc. Then after two weeks they messaged me that your account was open now. But still, I was not able to login. I email them again and then they replied (after 2 more weeks) that now it's open. So, in my case, it takes almost 2 months to get my account open.

**Reply by yichen-fjy** [reply]
> Bro same happened to me like 3 months ago. They suspended my account without any warning or mail. I contacted them on support, and they reply after almost 40 days and says that your name matches some name on US Sanctions List (I was WT* 😂) and then they told me to provide picture of national identity card etc. Then after two weeks they messaged me that your account was open now. But still, I was not able to login. I email them again and then they replied (after 2 more weeks) that now it's open. So, in my case, it takes almost 2 months to get my account open.

My God😂, it's taking so long! My friend must have been upset by the wait.

**Comment by yichen-fjy** [comment]
Hi Lili, @queenofcorgis, Can you help me reconfirm that the ticket is in the right place? The earliest ticket's number is 1989881. My friend @zonemeen has been waiting for more than two weeks. Thank you so much!

**Reply by queenofcorgis** [reply]
Hi there it is in the right place - there are still tickets ahead of it so please hang tight!

**Comment by ejrv** [comment]
Maybe it got hacked or something...

**Reply by cunzhang2** [reply]
I hope that's not the case. Have you had any experiences with suspended accounts like this?
